lib/tedious.js

```diff
@@ -193,6 +193,14 @@ const parameterCorrection = function (value) {
 class ConnectionPool extends base.ConnectionPool {
   _poolCreate () {
     return new base.Promise((resolve, reject) => {
+      const resolveOnce = (v) => {
+        resolve(v)
+        resolve = reject = () => {}
```

Comment:
nitpick: functionally i think this is valid, although most lint rules recommend avoiding mutation of function parameters.

Reply:
I know, but it's not in the `standard` code linting.... 😆 

I could just mutate the `resolveOnce` vars instead, but that just seems to be jumping through hoops for no real reason